examples/platform/nxp/k32w/k32w0/scripts/detokenizer.py

```diff
@@ -53,7 +53,7 @@ def decode_string(tstr, detok):
         if s.find('$') == 0:
             return None
         return s
-    except:
+    except ValueError:
```

Comment:
I strongly suggest doing the fixes by area, not by fixup type.  This change needs review from someone familiar with this specific script, for example....